lib/TXG/Utils.hs

```diff
@@ -135,7 +135,7 @@ cidFromText t = case readEither (T.unpack t) of
 
 chainIds :: ChainwebVersion -> [ChainId]
 chainIds Mainnet01 = ChainId <$> [0..9]
-chainIds Development = ChainId <$> [0..9]
+chainIds Development = ChainId <$> [0..19]
```

Comment:
Is there some way we can make this automatically detect the number of chains?  It would be nice to not have to worry about changing this in the future.

Reply:
In theory, yes, but I am not sure if it's worth the dev time. 

Ideally, chainweb-api, should provide all information about chainweb versions and all other tools (possibly even chainweb-node) should get it from there.